src/data/movie.hpp

```diff
@@ -24,7 +24,6 @@
 namespace rigel { namespace data {
 
 struct MovieFrame {
-  MovieFrame() = default;
```

Comment:
This works, though the way I fixed it was to add the default constructor to image.hpp:

```
class Image {
public:
  Image();
  Image(PixelBuffer&& pixels, std::size_t width, std::size_t height);
  Image(const PixelBuffer& pixels, std::size_t width, std::size_t height);
  Image(std::size_t width, std::size_t height);
```

(I'm no cpp guy though, so I'll defer to you; just wanted to point out how I fixed it)